frontend/packages/kubevirt-plugin/src/selectors/vm-template/selectors.ts

```diff
@@ -141,7 +141,7 @@ export const getRelevantTemplates = (
   workloadProfile: string,
   flavor: string,
 ) => {
-  if (!commonTemplates || commonTemplates.length === 0 || !os || !workloadProfile) {
+  if (!commonTemplates || commonTemplates.length === 0) {
```

Comment:
`(commonTemplates || [])` bellow will do the same job

Reply:
:exclamation: Nice, thanks fixing

Reply:
fixed